homework/style.css

```diff
@@ -1,3 +1,68 @@
+*{
+  padding: 0;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+.root{
+  display: flex;
+  flex-wrap: wrap;
+}
 .alert-error {
+  background-color: #f4f4f4;
   color: red;
+  font-size: 1.2em;
+}
+.title {
+  margin: 0 5%;
+  padding: 15px 10px;
+  background-color: rgb(80, 102, 231);
+  color: white;
+  font-size: 1em;
+}
+ul {
+  list-style: none;
+  margin: 1% 5%;
+  box-shadow: 1px 2px 3px 1px #666;
+}
+ul li {
```

Comment:
In general, you avoid tag names as a target for styling, because they will be applied globally throughout your HTML document. As more content gets added to your HTML page, this can quickly cause issues. Use CSS classes instead.